### DISCUSSION TITLE
自定义校验的应用

### DISCUSSION THREAD

**Discussion by happyElina** [alibaba/formily#1760] [Q&A 2.x] [answered]
import { registerValidateRules } from '@formily/core'

registerValidateRules({
  custom(value) {
    return value > 100 ? 'error' : ''
  },
})

如此注册的一个自定义校验，在schema参数里怎么传递呢？
https://react.formilyjs.org/api/shared/schema 中的参数列表里，x-validator 和 format 的参数类型都指向了一个校验类型声明，这块不是很明白。。。是没法这样registerValidateRules直接在schema参数里用字符串‘custom’来用吗？必须用markup 那种方式来用吗？用markup的话要怎么用呢？
ps: 好像没有关于自定义校验这块的使用demo

**Comment by janryWang** [comment, marked as answer]
x-validator:{custom:true}
